README.md

```diff
@@ -1,39 +1,156 @@
+coursesimgroup. After you have successfully installed the server login to the mysQl command  using the root password and carry out the commands (press enter after each semicolon). To install on mac, first, go to https://dev.mysql.com/downloads/mysql/ or search MySQL Community Server.
+
+Choose the operating system as MacOS, then select the OSVersion. If you are using MacBook M1 or M2, choose the ARM, if you are using MacBook Intel, Choose x86.
+
+<img width="347" alt="image" src="https://github.com/CSC207-2023Y-UofT/course-project-held-together-by-duct-tape/assets/133050886/2d96e065-58d0-4b5a-8d2d-ca48f257c425">
+
+Second, open the download package, (if the Mac says the program can not be authenticated， just open the setting->Security and privacy->allow the file to install.)
+After installation is successful, if you did everything in the instructions, you can open the setting on MacBook now, and you will see a cute dolphin and two green dots.
+Finally, activate your MySQL. Go to terminal to type: mysql -u root -p
+If there is an error like: “zsh:command not found: mysql”
+Just type: alias mysql=/user/local/mysql/bin/mysql
+Then re-enter: mysql -u root -p, enter your password (coursesimgroup).
+Tip, you can use terminal to operate MySQL, or you can choose to download MySQL Workbench(to download workbench are pretty easy, just go to the MySQL website and choose your version of workbench, install it as other normal apps.)
+
+<img width="290" alt="image" src="https://github.com/CSC207-2023Y-UofT/course-project-held-together-by-duct-tape/assets/133050886/62a0ffd5-a552-4727-a456-ca0101e7c13a">
+
+## Commands (In Order):
+
+- Create database course_simulator;
+
+- use course_simulator;
+
+- create table students (StudentID Varchar(25), Password Varchar(25), CourseID Varchar(25), CourseGrade float);
+
+- create table courses (CourseID Varchar(25), PrerequisiteCourseID Varchar(25), PrerequisiteGrade float, Question Varchar(250), Answer Varchar(250), Points int);
+
+- create table sessioncourse (CourseID Varchar(25), Question Varchar(250), Answer Varchar(250), Points int);
+
+- create table sessionstudent (StudentID Varchar(25), CourseID Varchar(25), CourseGrade float);
+
+- Show tables;  (use this command to confirm that the above tables were made successfully!)
+
+![image](https://github.com/CSC207-2023Y-UofT/course-project-held-together-by-duct-tape/assets/133050886/e2a3341e-1f98-408f-a854-be762390c056)
+
+
+This is a course simulator in which the user can pick between an instructor mode and a student mode. 
+
+
+<img width="392" alt="image" src="https://github.com/CSC207-2023Y-UofT/course-project-held-together-by-duct-tape/assets/133050886/c7615b93-334d-4bbd-a902-bc55b4c786c1">
+
+
+In instructor mode, the user can create a course. They must give a course code, prerequisites, questions, answers and their corresponding number of points. Then that specific course is saved. 
+
```

Comment:
Once the instructor click the 'Create' button, the input will be checked, if all the features fit the requirements, then this specific course will be saved. Otherwise, here will show up an error message and the create course screen will be clean. The instructor have to start a complelety new  create course screen. 

If the course be saved successfully, then the instructor can not review or change this course in instructor mode any more.